tests/integration/renderer/test_loading_states.py

```diff
@@ -167,3 +167,111 @@ def find_text(spec):
 
     find_spinners()
     find_text({1: 1, 2: 1, 3: 1, 4: 1})
+
+
+def test_update_title_default(dash_duo):
```

Comment:
```suggestion
def test_rdls003_update_title_default(dash_duo):
```

`rdls` comes from the path of this file, renderer->loading_states, and `003` is serial within the file. This lets you run this whole file `pytest -k rdls` or just this test case `pytest -k rdls003`